cluster/gce/ubuntu/master.yaml

```diff
@@ -1,128 +0,0 @@
-      ExecStartPre=/bin/mount --bind /home/kubernetes/bin /home/kubernetes/bin
-      ExecStartPre=/bin/mount -o remount,exec /home/kubernetes/bin
-      ExecStartPre=/usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error	-H "X-Google-Metadata-Request: True" -o /home/kubernetes/bin/configure.sh http://metadata.google.internal/computeMetadata/v1/instance/attributes/configure-sh
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/configure.sh
-      ExecStart=/home/kubernetes/bin/configure.sh
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-master-configuration.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Configure kubernetes master
-      After=kube-master-installation.service
-
-      [Service]
-      Type=oneshot
-      RemainAfterExit=yes
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/configure-helper.sh
-      ExecStart=/home/kubernetes/bin/configure-helper.sh
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-docker-monitor.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes health monitoring for docker
-      After=kube-master-configuration.service
-
-      [Service]
-      Restart=always
-      RestartSec=10
-      RemainAfterExit=yes
-      RemainAfterExit=yes
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/health-monitor.sh
-      ExecStart=/home/kubernetes/bin/health-monitor.sh docker
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kubelet-monitor.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes health monitoring for kubelet
-      After=kube-master-configuration.service
-
-      [Service]
-      Restart=always
-      RestartSec=10
-      RemainAfterExit=yes
-      RemainAfterExit=yes
-      ExecStartPre=/bin/chmod 544 /home/kubernetes/bin/health-monitor.sh
-      ExecStart=/home/kubernetes/bin/health-monitor.sh kubelet
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-logrotate.timer
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Hourly kube-logrotate invocation
-
-      [Timer]
-      OnCalendar=hourly
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kube-logrotate.service
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes log rotation
-      After=kube-master-configuration.service
-
-      [Service]
-      Type=oneshot
-      ExecStart=-/usr/sbin/logrotate /etc/logrotate.conf
-
-      [Install]
-      WantedBy=kubernetes.target
-
-  - path: /etc/systemd/system/kubernetes.target
-    permissions: 0644
-    owner: root
-    content: |
-      [Unit]
-      Description=Kubernetes
-
-      [Install]
-      WantedBy=multi-user.target
```

Comment:
This is needed by Ubuntu.

The GCI image doesn't have this in its systemd services because GCI uses systemd as a delegate (pid 1 is upstart).

Therefore symlinking ubuntu -> gci does not work: if a node reboots the services won't be brought back up.

That is the reason I originally duplicated some of the files in the subdirectory instead of symlinking, the node.yaml and master.yaml files need to be different.